dart-impl/mpi/src/dart_mpi_op.c

```diff
@@ -0,0 +1,117 @@
+
+#include <mpi.h>
+
+#include <dash/dart/if/dart_types.h>
+#include <dash/dart/mpi/dart_communication_priv.h>
+#include <dash/dart/base/assert.h>
+
+typedef void (*dart__mpi_min_max_reduce_t)(void*, void*, int*, MPI_Datatype*);
+
+#define DART_NAME_MINMAX_OP(__name) dart__mpi_min_max_reduce_##__name
+
+#define DART_DECLARE_MINMAX_OP(__name) \
+DART_INLINE void DART_NAME_MINMAX_OP(__name)(            \
+  void *lhs_, void *rhs_, int *len, MPI_Datatype *dptr)
```

Comment:
Sorry, I didn't have that in my version either. We can make the lhs `const` here and below.

Reply:
Technically we could but the MPI standard does not have the `const` in it's interface so we'd have to do some nasty casts when passing the function to MPI to suppress compiler warnings.

Reply:
Of course we can make `lhs` inside the function body `const`, didn't think of that initially (not sure that that changes anything